tools/PI/DevHome.PI/BarWindowHorizontal.xaml.cs

```diff
@@ -0,0 +1,337 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
All the duplicated code in BarWindowHorizonal.xaml.cs and BarWindowVertical.xaml.cs is a great reason to not have code here, and use a BarWindowViewModel instead. We strongly recommend using MVVM practices.
(After I wrote this I saw your comment about moving things to the view model)

Reply:
Great minds think alike. :)